recipes/newrelic/infrastructure/kubernetes.yml

```diff
@@ -153,8 +153,8 @@ install:
             exit 131
           fi
 
-          if [[ "$SERVER_MAJOR_VERSION" -lt 1 ]] || [[ "$SERVER_MAJOR_VERSION" -eq 1 && "$SERVER_MINOR_VERSION" -lt 16 ]] || [[ "$SERVER_MAJOR_VERSION" -eq 1 && "$SERVER_MINOR_VERSION" -gt 24 ]]; then
-            echo "Installation failed. Kubernetes v1.16 to v1.24 is required, found v${SERVER_MAJOR_VERSION}.${SERVER_MINOR_VERSION}" >&2
+          if [[ "$SERVER_MAJOR_VERSION" -lt 1 ]] || [[ "$SERVER_MAJOR_VERSION" -eq 1 && "$SERVER_MINOR_VERSION" -lt 16 ]] || [[ "$SERVER_MAJOR_VERSION" -eq 1 && "$SERVER_MINOR_VERSION" -gt 25 ]]; then
+            echo "Installation failed. Kubernetes v1.16 to v1.25 is required, found v${SERVER_MAJOR_VERSION}.${SERVER_MINOR_VERSION} (note: minikube version 1.28.0 only supports kubernetes versions up to 1.25.3)" >&2
```

Comment:
sorry can we update the comment to say (note: if you're running minikube, make sure your version of minikube supports the version of kubernetes that you're on by running `minikube config defaults kubernetes-version`)

Reply:
that way we dont have to update this message every time there is a new release.

Reply:
actually please disregard the addition of any notes.

Reply:
and revert to https://github.com/newrelic/open-install-library/pull/838/commits/60abdd75a770ca5198cf803918aae33d64354128